packages/qwik/src/optimizer/core/src/snapshots/qwik_core__test__example_lightweight_functional.snap

```diff
@@ -9,7 +9,7 @@ snapshot_kind: text
 
 import { $, component$ } from '@qwik.dev/core';
 
-export const Foo = component$(({color}) => {
+export const Foo = component$((props) => {
```

Comment:
This one was wrong actually, maybe we want to use some props? Not sure if important

Reply:
What do you mean? Later we use `...props` and never the `color`, so I think now it is correct?
or do you mean that this error is intentional?

Reply:
No it was destructuring color but using props

Reply:
yeah, and it is wrong, right? or should I revert it?